packages/ui5-application-writer/templates/optional/typescript/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "outDir": "./dist",
     "baseUrl": "./",
     "paths": {
-      "<%- app.id %>/*": [
+      "<%=app.id.replace(/\./g, '/') %>/*": [
```

Comment:
Should we keep `<%-` here? @IainSAP wasn't that the recommendation for the templates?
https://stackoverflow.com/a/11025348